index.html

```diff
@@ -7,8 +7,24 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
+  <link  rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap">
```

Comment:
This work but I suggest use @import word on scss file 

@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap');

other thing , what weights do you need only 300 and 400? think about it